settings.h

```diff
@@ -631,6 +631,7 @@ typedef union {
         uint16_t machine_position   :1,
                  buffer_state       :1,
                  line_numbers       :1,
+                 distance_to_go     :1,
```

Comment:
Adding this in the middle of the value will force a settings revision and a settings reset on update, add it after `when_homing` instead to avoid that.